xml/System/DateTime.xml

```diff
@@ -2033,7 +2033,7 @@ For applications in which portability of date and time data or a limited degree
 
  The fractional part of `value` is the fractional part of a day. For example, 4.5 is equivalent to 4 days, 12 hours, 0 minutes, 0 seconds, 0 milliseconds, and 0 ticks.
 
- The `value` parameter is rounded to the nearest millisecond.
+ The `value` parameter is no longer rounded to the nearest millisecond (as it used to be before .NET 7.0).
```

Comment:
```suggestion
In .NET 6 and earlier versions only, the `value` parameter is rounded to the nearest millisecond.
```

Reply:
@gewarren Thank you, I'll update the pattern on all cases